### DISCUSSION TITLE
how to use the new version epkg with borg

### DISCUSSION THREAD

**Discussion by capfredf** [emacscollective/epkg#37] [Support] [answered]
Hi  @tarsius, 

In the latest version,  the elisp files of epkg have moved into the directory `lisp/` and it looks like I have to manually set the load path for epkg to make `epkg-*` functions/commands available.

I use borg as my only package manager and it has been brought up to date.

**Comment by tarsius** [comment, marked as answer]
You shouldn't have to do that but you might have to restart Emacs.

**Reply by capfredf** [reply]
Restarting Emacs doesn't help either.

**Reply by tarsius** [reply]
This is my configuration for `epkg` in `.gitmodules` (i.e. the default):

```
[submodule "epkg"]
	path = lib/epkg
	url = git@github.com:emacscollective/epkg.git
```

Does yours differ?

And this is the code that determines the `load-path`:

```elisp
(defun borg-load-path (clone)
  "Return the `load-path' for the clone named CLONE."
  (let ((repo (borg-worktree clone))
        (path (borg-get-all clone "load-path")))
    (if  path
        (mapcar (lambda (d) (expand-file-name d repo)) path)
      (let ((elisp (expand-file-name "elisp" repo))
            (lisp (expand-file-name "lisp" repo)))
        (list (cond ((file-exists-p elisp) elisp)
                    ((file-exists-p lisp) lisp)
                    (t repo)))))))
```

You could try figuring out what is going on by adding debug statements to that function.
